src/library/Uno.Themes.WinUI.Markup/Theme/TextBox.cs

```diff
@@ -0,0 +1,201 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "TextBoxDeleteButtonForegroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("TextBoxDeleteButtonForegroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextBoxDeleteButtonForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("TextBoxDeleteButtonForegroundDisabled", true);
+					}
+				}
+
+				public static class Filled
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxForeground")]
+						public static ResourceValue<Brush> Default => new("FilledTextBoxForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FilledTextBoxForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxForegroundFocused")]
+						public static ResourceValue<Brush> Focused => new("FilledTextBoxForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledTextBoxForegroundDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxPlaceholderForeground")]
+						public static ResourceValue<Brush> Placeholder => new("FilledTextBoxPlaceholderForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxPlaceholderForegroundPointerOver")]
+						public static ResourceValue<Brush> PlaceholderPointerOver => new("FilledTextBoxPlaceholderForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxPlaceholderForegroundFocused")]
+						public static ResourceValue<Brush> PlaceholderFocused => new("FilledTextBoxPlaceholderForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxPlaceholderForegroundDisabled")]
+						public static ResourceValue<Brush> PlaceholderDisabled => new("FilledTextBoxPlaceholderForegroundDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForeground")]
+						public static ResourceValue<Brush> DeleteButton => new("FilledTextBoxDeleteButtonForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForegroundPointerOver")]
+						public static ResourceValue<Brush> DeleteButtonPointerOver => new("FilledTextBoxDeleteButtonForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForegroundFocused")]
+						public static ResourceValue<Brush> DeleteButtonFocused => new("FilledTextBoxDeleteButtonForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForegroundDisabled")]
+						public static ResourceValue<Brush> DeleteButtonDisabled => new("FilledTextBoxDeleteButtonForegroundDisabled", true);
```

Comment:
The should be moved to a `DeleteButtonForeground` class within the `Filled` class